COPYRIGHT.txt

```diff
@@ -62,40 +62,34 @@ Copyright: 2006-2020, assimp team
  2014-2021, Godot Engine contributors.
 License: BSD-3-clause
 
-Files: ./platform/android/java/lib/aidl/com/android/vending/licensing/
+Files: ./platform/android/java/lib/aidl/com/android/*
  ./platform/android/java/lib/res/layout/status_bar_ongoing_event_progress_bar.xml
- ./platform/android/java/lib/src/com/google/android/vending/expansion/downloader/
- ./platform/android/java/lib/src/com/google/android/vending/licensing/
+ ./platform/android/java/lib/src/com/google/android/*
  ./platform/android/java/lib/src/org/godotengine/godot/input/InputManagerCompat.java
  ./platform/android/java/lib/src/org/godotengine/godot/input/InputManagerV16.java
 Comment: The Android Open Source Project
-Copyright: 2008-2013, The Android Open Source Project
+Copyright: 2002, Google, Inc.
+Copyright: 2008-2016, The Android Open Source Project
```

Comment:
I don't think they can be two `Copyright:` sections in the Debian machine-readable changelog spec: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/

But this could be:
```
Copyright: 2008-2016, The Android Open Source Project
 2002, Google, Inc.
```
as done for other `Files:` with multiple copyright statements.

Reply:
Done.